RELEASES.md

```diff
@@ -94,17 +90,14 @@ A MAJOR version of ICS will always be backwards compatible with the previous MAJ
 
 The following table indicates the compatibility of currently active releases:
 
-| Consumer | Provider | `v2.0.0` | `v2.4.0-lsm` | `v3.1.0` | `v3.2.0` | `v3.3.0` | `v4.0.0` |
-|----------|----------|----------|--------------|----------|----------|----------|----------|
-| `v2.0.0` || ✅ | ✅ | ✅ (1) | ✅ (1) | ✅ (1),(2) | ❌ |
-| `v3.1.0` || ✅ | ✅ | ✅ | ✅ | ✅ (2) | ❌ |
-| `v3.2.0` || ✅ | ✅ | ✅ | ✅ | ✅ (2) | ✅ |
-| `v3.3.0` || ✅ (2) | ✅ (2) | ✅ (2) | ✅ (2) | ✅ | ✅ |
-| `v4.0.0` || ✅ (2) | ✅ (2) | ✅ (2) | ✅ (2)| ✅ (2)| ✅ |
+| Consumer | Provider | `v3.2.0` | `v3.3.0` | `v3.3.3-lsm` | `v4.0.0` |
+|----------|----------|----------|----------|--------------|----------|
+| `v3.2.0` || ✅ | ✅ (1) | TBA | ✅ |
+| `v3.3.0` || ✅ (1) | ✅ | TBA | ✅ |
+| `v4.0.0` || ✅ (1)| ✅ (1)| TBA | ✅ |
```

Comment:
@bermuell could you please confirm that v3.3.3-lsm is compatible with all these versions of the consumer?